packages/x-components/tests/unit/main-scroll.spec.ts

```diff
@@ -166,6 +169,44 @@ describe('testing MainScroll component', () => {
         });
       });
 
+      it('restores the scroll with transitions enabled', () => {
+        const template = `
```

Comment:
If you are going to use a variable only once, usually it is better to inline it in the function that uses it.